README.md

```diff
@@ -163,6 +163,7 @@ logging config format for content details.
 -  Full volume name with format like "volume@datastore" cannot be specified in the compose file for stack deployment. [#1315](https://github.com/vmware/docker-volume-vsphere/issues/1315). It is a docker compose issue and a workaround has been provided in the issue.
 -  Specifying "Datastore Cluster" name during volume creation is not supported. Datastore clusters (as a part of Storage DRS) is a VC feature and not available on individual ESX. [#556](https://github.com/vmware/docker-volume-vsphere/issues/556)
 -  Volume creation using VFAT filesystem is not working currently. [#1327](https://github.com/vmware/docker-volume-vsphere/issues/1327)
+-  Volume is not shown when running ```docker volume ls``` and ```vmdkops_admin volume ls``` command after performing storage vMotion of VM when volume is attached to VM. It is an expected behavior since storage vMotion messes up the location and names of attached vmdk files. [#1618] (https://github.com/vmware/docker-volume-vsphere/issues/1618)
```

Comment:
Agree, should not mention _messes up_. Actually are we even able to svmotion when attaching the volume in the default independent mode?

The wording should mention exactly the scenarios where this behavior is observed - volumes attached as independent vs. persistent,  impact when the volumes are not attached vs. attached and how to avoid the issue (address volume via qualified name  - vol@ds - for example).